Makefile

```diff
@@ -41,13 +41,13 @@ coverage: ## check code coverage quickly with the default Python
 		--cov-report term-missing \
 		--cov=django_sorcery tests \
 		--doctest-modules \
-		django_sorcery tests
+		tests django_sorcery
```

Comment:
tests initialize django so it needs to be first. or need to use pytest-django

Reply:
:+1: